arrow/test/test_arrow.py

```diff
@@ -173,11 +179,38 @@ def test_compare_runtime():
     print('reference time elapsed: {}'.format(reference_end - reference_start))
     print('obsidian time elapsed: {}'.format(obsidian_end - obsidian_start))
 
+def test_memory():
+    stoichiometric_matrix, rates, initial_state, final_state = load_complexation()
+    duration = 1
+    amplify = 100
+
+    this = psutil.Process(os.getpid())
+    memory = 0
+    memory_previous = 0
+
+    system = StochasticSystem(stoichiometric_matrix, rates, random_seed=np.random.randint(2**31))
+    obsidian_start = time.time()
+    for i in range(amplify):
+        memory = this.memory_info().rss
+        if (memory != memory_previous):
+            print('memory increase iteration {}: {}'.format(i, memory))
+            memory_previous = memory
+
+        result = system.evolve(duration, initial_state)
+        difference = np.abs(final_state - result['outcome']).sum()
+
+        print('difference is {}'.format(difference))
```

Comment:
The difference in what?

Reply:
This reminds me of an old joke: 

"What is the difference between a duck?"